app/values/iiif_path.rb

```diff
@@ -9,6 +9,10 @@ def to_s
     "#{iiif_url}/#{escaped_path}"
   end
 
+  def thumbnail
+    "#{self}/full/100,/0/default.jpg"
```

Comment:
Shouldn't this be 200 or 200,150 like we're currently using?

https://github.com/projecthydra-labs/curation_concerns/blob/master/curation_concerns-models/app/models/concerns/curation_concerns/file_set/derivatives.rb#L40-L41